R/get_plan_doc_info.R

```diff
@@ -0,0 +1,36 @@
+#' Get planning doc info
```

Comment:
```suggestion
#' Get information for planning document
```
Just to match the proposed `get_plan_doc_info` title ;)